src/main/kotlin/com/seokjoo/todo/common/exception/TodoExceptionType.kt

```diff
@@ -40,4 +40,12 @@ enum class TodoExceptionType(
     ),
     AUTH_REFRESH_TOKEN_NOT_VALID(message = "유효하지 않은 토큰입니다.", errorCode = "A003_TOKEN_NOT_VALID", httpStatusCode = 401),
     AUTH_SIGN_UP_ERROR(message = "이미 존재하는 ID 입니다", errorCode = "A004_USER_ALREADY_EXIST", httpStatusCode = 400),
+
+    // MONEY
+    BALANCE_NOT_ENOUGH_MONEY(message = "잔액 부족입니다.", errorCode = "M000_BALANCE_NOT_ENOUGH_MONEY", httpStatusCode = 400),
```

Comment:
```suggestion
    BALANCE_NOT_ENOUGH(message = "잔액 부족입니다.", errorCode = "M000_BALANCE_NOT_ENOUGH", httpStatusCode = 400),
```

매우매우 사소한 의견인데, `MONEY` 없어도 의미 충분히 잘 전달되는 느낌이에유!

Reply:
[d9406fa](https://github.com/Ahn-seokjoo/todo-spring/pull/58/commits/d9406fac55cdc7eb0b72f1cd175ffa0c1b208c0e) 조아요 반영 !